triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/PrettyPrint.scala

```diff
@@ -152,7 +152,7 @@ object PrettyPrint {
       str(value)
 
     case SValue.SText(value) =>
-      text(value)
+      text(s"$value")
```

Comment:
Bug fix: `value` can sometimes be `null`, so we first stringyfy the `null` value before letting the pretty print get the value.